tests/jobs/svn-checkout-update.sh

```diff
@@ -11,6 +11,11 @@ fi
 
 case $state in
     (0) echo "::Create repo. -- continue -- updated::"
+    svnadmin create svn-repo
+    svn co file://$PWD/$repo svn-checkout
+    touch svn-checkout/svn-file
+    svn add svn-checkout/svn-file
+    svn commit -m "add svn file" svn-checkout/svn-file
```

Comment:
Is this needed because it is no longer sharing a state space with a previously ordered svn test?

Reply:
Correct (same with the `bzr-checkout-update.sh` script) -- or at least, because the `bzr-checkout-update` test can be run before `bzr` now (unsure if this is always the case).